### DISCUSSION TITLE
Quando creo una fattura non salva l'indirizzo email del cliente

### DISCUSSION THREAD

**Discussion by christiancannata** [fattureincloud/api#360] [Q&A] [answered]
Sto provando a creare una fattura tramite API e sembra non salvare l'indirizzo email nonostante venga passato come valore nella richiesta JSON:

- Sto utilizzando la libreria PHP con il metodo setEmail
- Nel json body c'è correttamente il campo compilato
- Non viene visualizzato (penso quindi nemmeno salvato) in fase di chiamata API

POST  https://api-v2.fattureincloud.it/c/616575/issued_documents

{
  "data": {
    "entity": {
      "name": "Cappata Christian",
      "vat_number": "XXXX",
      "tax_code": "XXXX",
      "address_street": "XXXX",
      "address_postal_code": "XXXX",
      "address_city": "San Vito al Tagliamento",
      "address_province": "MI",
      "country": "Italia",
      "email": "XXXX@gmail.coma",
      "certified_email": ""
    },
    "type": "invoice",
    "currency": {
      "id": "EUR"
    },
    "language": {
      "code": "it",
      "name": "italiano"
    },
    "subject": "Ordine n.5297",
    "visible_subject": "Ordine n.5297",
    "payment_method": {
      "id": "2122109"
    },
    "e_invoice": true,
    "ei_data": {
      "payment_method": "MP01"
    },
    "items_list": [
      {
        "name": "Promuovere le competenze di autonomia e di autodeterminazione nei Disturbi dello Spettro dell'Autismo",
        "qty": 1,
        "net_price": "300",
        "vat": {
          "id": "5897625"
        }
      }
    ],
    "payments_list": [
      {
        "due_date": "2023-09-28",
        "amount": 240,
        "status": "paid",
        "payment_account": {
          "id": 552232
        },
        "paid_date": "2023-09-28"
      }
    ]
  },
  "options": {
    "fix_payments": true
  }
}

[Screenshot errore](https://i.ibb.co/rFbbF8Q/Screenshot-2023-10-03-alle-15-28-09.png)

**Comment by emajo** [comment, marked as answer]
Ciao @christiancannata questo argomento è stato affrontato già numerose volte all'interno della community, in particolare le consiglio di leggere [questa discussion](https://github.com/fattureincloud/api/discussions/226).

Grazie,
Emanuele.

**Comment by christiancannata** [comment]
Grazie mille!
